src/app/song/songs.service.ts

```diff
@@ -16,7 +16,7 @@ export class SongService{
                 private notificationService:NotificationPopupService){}
 
     getSongs(){
-        this.Http.get<{message: string; songs: any}>('http://localhost:3000/api/getSongs')
+        this.Http.get<{message: string; songs: any}>('http://localhost:3000/api/songs')
```

Comment:
Put the base endpoint in some global variable in some consts file so if someone wants to run it on another port they won't have to change every single URL you've sent a request to.